providers/one-signal/src/lib/one-signal.provider.ts

```diff
@@ -29,7 +29,12 @@ export class OneSignalPushProvider implements IPushProvider {
     const { sound, badge, ...overrides } = options.overrides ?? {};
 
     const res = await this.oneSignal.createNotification({
-      include_player_ids: options.target,
+      ...(overrides['one-signal']?.externalUserIds !== undefined
+        ? {
+            include_external_user_ids: overrides['one-signal'].externalUserIds,
+            channel_for_external_user_ids: 'push',
+          }
+        : { include_player_ids: options.target }),
```

Comment:
```suggestion
   const { externalUserIds } =  overrides['one-signal'] || {};
   const res = await this.oneSignal.createNotification({ 
      ...(externalUserIds !== undefined
        ? {
            include_external_user_ids: externalUserIds,
            channel_for_external_user_ids: 'push',
          }
        : { include_player_ids: options.target }),
```
We can reuse the extraction of the `externalUserIds`.
Any way of doing it should be enough:
```
   const externalUserIds =  overrides['one-signal']?.externalUserIds;
```